modules/database/src/main/java/org/eclipse/xpanse/modules/database/service/DeployServiceEntity.java

```diff
@@ -88,4 +93,14 @@ public class DeployServiceEntity extends CreateModifiedTime {
     @Cascade({CascadeType.ALL})
     private List<DeployResourceEntity> deployResourceList;
 
+    /**
+     * The properties of the deployed service.
+     */
+    @ElementCollection
+    @CollectionTable(name = "DEPLOY_SERVICE_PROPERTY",
+            joinColumns = @JoinColumn(name = "DEPLOY_SERVICE_ID", nullable = false))
+    @MapKeyColumn(name = "P_KEY")
+    @Column(name = "P_VALUE")
+    private Map<String, String> property;

```

Comment:
instead of just "property".. can the field be name deployedServiceProperties?

Reply:
Ok.